docs/source/audio_process.mdx

```diff
@@ -8,17 +8,26 @@
 
 ## Installation
 
-The [`Audio`] feature is an experimental feature and should be installed as an extra dependency in 🤗 Datasets. Install the [`Audio`] feature with pip:
+The [`Audio`] feature is an experimental feature and should be installed as an extra dependency in 🤗 Datasets. Install the [`Audio`] feature (and its dependencies) with pip:
```

Comment:
Maybe we should remove the "experimental feature" part since the Audio feature has been in the library for some time now.

Reply:
Thanks. Done.